src/directives/NgGrid.ts

```diff
@@ -120,17 +127,23 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 				private _ngEl: ElementRef,
 				private _renderer: Renderer,
 				private componentFactoryResolver: ComponentFactoryResolver,
-				private _containerRef: ViewContainerRef) {}
+				private _containerRef: ViewContainerRef) {
+
+					this._defineListeners();
+				}
 
 	//	Public methods
 	public ngOnInit(): void {
+		this._enableResizeListeners();

```

Comment:
I guess this listener doesn't depend on any condition, let me know if my premise is wrong.